metashare/api/serializers.py

```diff
@@ -108,6 +109,7 @@ class Meta:
             "currently_creating_pr",
             "pr_url",
             "pr_is_open",
+            "assigned_users",
```

Comment:
This is causing an error: `TypeError: Object of type Hashid is not JSON serializable`